Formula/h2c.rb

```diff
@@ -0,0 +1,19 @@
+class H2c < Formula
+  desc "Headers 2 curl"
+  homepage "https://curl.se/h2c/"
+  url "https://github.com/curl/h2c/archive/fee2a0361b4124dbd5565cee2a90ffb0c80ab7b0.tar.gz"
```

Comment:
`h2c` does not have any tagged releases, so I don't think this formula is acceptable in Homebrew/core. See: https://docs.brew.sh/Acceptable-Formulae#stable-versions

You can try asking upstream if they're willing to tag their releases or host this formula in your own tap.

If you choose to do the latter, here are some docs to help you get started:

https://docs.brew.sh/Taps
https://docs.brew.sh/Interesting-Taps-and-Forks
https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap
https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

Reply:
Thanks for the tip! :) I'll ask the upstream.